cpp/tests/io/parquet_test.cpp

```diff
@@ -3346,56 +3373,23 @@ TEST_F(ParquetWriterTest, CheckPageRows)
   cudf_io::write_parquet(out_opts);
 
   // check first page header and make sure it has only 5000 values
```

Comment:
This magic number took me a moment to find, out of context. The variable name is more meaningful imo.
```suggestion
  // check first page header and make sure it has only page_rows values
```